bin/submit-task

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## This script submits new tasks for SAPS interacting with Dispatcher backend.
+
+## Checking args
+if [ $# -ne 12 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD LOWER_LEFT_LATITUDE UPPER_RIGHT_LATITUDE LOWER_LEFT_LONGITUDE UPPER_RIGHT_LONGITUDE PROCESSING_INIT_DATE PROCESSING_FINAL_DATE INPUTDOWNLOADING_TAG PREPROCESSING_TAG PROCESSING_TAG DISPATCHER_ACCESS_IP"
+  echo "e.g. $0 <user-email> <user-paswword> -7.913 -6.547 -37.814 -35.757 2015-06-23 2015-06-23 googleapis default ufcg-sebal <dispatcher-access-ip>"
```

Comment:
remove echo from line 9

Reply:
... and, add it as an example in your end-to-end test

Reply:
Done